kubernetes/setup-environment

```diff
@@ -11,8 +11,16 @@ done
 
 log "MariaDB container is ready"
 log "Setting up database"
-docker exec -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup
-docker exec -e RAILS_ENV=test -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup
+until docker exec -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup;
+do
+    log "Waiting for mariadb to accept connections"
+    sleep 5
+done
+until docker exec -e RAILS_ENV=test -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup;
```

Comment:
Maybe with the unix socket this is a bit different. There could be an execution path in which `/var/run/mysqld/mysql.sock` was created, but `mysql` wasn't listening yet there, might that be? I would say we keep the same strategy as before, listening on `127.0.0.1` instead of a unix socket, unless there are strong reasons to switch to a unix socket.